static/sass/patterns/_dynamic-title.scss

```diff
@@ -0,0 +1,78 @@
+// Dynamic Titles
+// ==============
+
+.dynamic-title {
+  .title-main {
+    margin-top: size('shim');
+  }
+}
+
+.title-select {
+  @include font-family('body');
+  display: inline-block;
+
+  .is-dropdown {
+    min-width: 100%;
+    transform: translateY(size('border-thin'));
+  }
+}
+
+.title-current {
+  border: size('border-thin') solid color('action');
+  border-radius: size('border-radius');
+  box-shadow: 0 0 5px color('shadow');
+  display: inline-block;
+  font-style: italic;
+  padding: 0 size('shim') size('quarter-shim');
+  transition: all 300ms ease-in-out;
+
+  #{$focus} {
+    background: color('background');
+    border-color: color('action-hover');
+    color: color('action-hover');
+  }
+
+  &[aria-pressed='true'] {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+}
+
+[name='title-option'] {
+  @include is-hidden;
+
+  &:checked {
+    ~ .dropdown-item {
+      font-style: italic;
+    }
+  }
+
+  &:focus {
+    ~ .dropdown-item {
+      color: color('action');
+    }
+  }
+}
+
+.tagline-option {
+  // float "isolation" keeps container height consistent…
+  float: left;
+  margin-right: -200%;
+  opacity: 0;
+  pointer-events: none;
+  position: relative;
+  transform: translate3d(0, -100%, 0);
+  transition: opacity 200ms ease-in-out, transform 300ms ease-in-out;
```

Comment:
we should maybe one day make a z-index like map for transitions

Reply:
@stacyk That's an interesting idea. Which of these are you imagining?

```
transitions: (
  'fast': 150ms, // only store times/easings
  'opacity-in': opacity 150ms ease-in-out, // store specific property/time/easing combos
  'slide-fade': (opacity 200ms ease-in-out, transform 300ms ease-in-out), // store full values
);

Seems like time/easing maps would be a great start - but I'm open to the other ideas if they also seem useful. We could allow both examples 2 and 3 without too much difficulty.

Reply:
Well, I think that we would have to look at our code to see what types of transitions are being repeated to see how much info we should include in the map. If we look at our code, I couldn't find many overlapping patterns except maybe background always seems to be 150ms/ease when transitioning.  I do wonder though if we established this, it would be something we try to use (I know when chaining animations the timing will often be custom to that event so we couldn't always refer to a map like we do our scale options).